flocx_market/common/policy.py

```diff
@@ -0,0 +1,165 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import itertools
+
+from oslo_policy import policy
+
+import flocx_market.conf
+
+CONF = flocx_market.conf.CONF
+_ENFORCER = None
+
+default_policies = [
+    policy.RuleDefault('is_admin',
+                       'role:admin or role:flocx_market_admin',
+                       description='Full API access'),
+]
+
+bid_policies = [
```

Comment:
If I'm reading this properly, the default policies are all admin-only, right? The idea being that an operator would always need to deploy a customized policy file to make the service generally available?

Reply:
Yes. I have decided this is the best way to preserve backwards compatibility with everyone using this code.

Reply:
I don't think there *is* anyone using this code :)

Reply:
what, no, I'm sure there are billions